### ISSUE TITLE
Performance: one training epoch exceeds 60 seconds with XGBoost-attention model on temporal data

### ISSUE THREAD

**Issue by nm-z** [nm-z/recipe-dev#619]
The public Recipe run remained active after 60 seconds for one epoch of training with model xgbst().silu().qi(4).0.attn(1).huber().qi(4).k.m.loss(bce) on data/temporal/series_sqlite.sqlite using fp(64) precision and adamw optimizer.

## Machine provenance

- Discoverer: Recipe exhaustive Cartesian traversal
- Failure fingerprint: `12e746bcbb00049f`
- Classifier provider: `opencode`
- Classifier model: `nemotron-3.5-lightning-free`
- Classifier effort: `default`

### Classifier decision

```json
{
  "verdict": "new",
  "issue": 0,
  "title": "Performance: one training epoch exceeds 60 seconds with XGBoost-attention model on temporal data",
  "body": "The public Recipe run remained active after 60 seconds for one epoch of training with model xgbst().silu().qi(4).0.attn(1).huber().qi(4).k.m.loss(bce) on data/temporal/series_sqlite.sqlite using fp(64) precision and adamw optimizer.",
  "rationale": "No existing GitHub issue represents the same earliest failing operation through the same implementation path. The model composition (XGBoost estimator followed by attention with 4-bit quantization and fp64 precision) has not been reported as a performance issue. The observed 60-second runtime for one epoch is materially slower than expected throughput for this workload and hardware configuration, where fp(64) precision on GPU typically provides lower throughput than fp32/fp16, and the XGBoost-attention combination has not been previously documented as a performance issue."
}
```

## Deterministic failure packet

kind=performance
id=12e746bcbb00049f
base=commit=9cc65a63b4677c53372379219cb280da9085ac1b tracked_tree=modified
cursor=seed:0 cursor:24347 next:24348 composition:64412691294433
expected=one training epoch uses throughput proportionate to its operations, data volume, memory traffic, and available hardware
observed=the public Recipe run remained active after 60 seconds
measurement=elapsed_seconds:60 backend:nv0
replay=the configured slow-runtime threshold was crossed stable:true
command=cargo run --bin recipe -- /home/nate/Desktop/recipe-machine-workload/target/debug/recipe-trials/recipe-trial-9243-24347/reproduction.rs
reproduction:
```rust
use recipe::*;

fn main() {
	let bundle = format!("/tmp/recipe-composition-repro-{}.ogdl", std::process::id());
	let data = recipe.data("data/temporal/series_sqlite.sqlite")
		.target("target")
		.broadcast();
	let model = recipe.model()
		.xgbst().silu().qi(4).0
		.attn(1).huber().qi(4).k.m
		.loss(bce);
	let report = recipe.train()
		.optimizer(adamw)
		.lr(0.001)
		.seed(64412691294433)
		.epochs(1)
		.log(all)
		.fp(64)
		.save(&bundle)
		.run(&model, &data);
	assert!(report.final_loss().is_finite());
}
```
backend=nv0

**Comment by nm-z** [comment]
Verification receipt: current `minimal` completes every #619 packet under 60 s.

- Base: `minimal` `10148e25` (release build on archy, public CLI `target/release/recipe --device nv0` pinned to one Tesla M60 die with `CUDA_VISIBLE_DEVICES`, cold = fresh native cache under a new HOME, warm = second run, `ulimit -v 16 GB`, measured with other agents active at the stated load averages)
- `issue-619.rs` on archy nv0 (Tesla M60, die 5): cold 31.73 s, warm 34.60 s; epoch 0.24 s; r2 -6.9515 bce 0.6437 xgbst.silu.Q4_0/attn.huber.Q4_K_M; bundle 633406 B; inference at width 512 = 22.742639896631495 (finite), SHA-256 unchanged; load 51.87/51.29
- Inference is a separate `recipe.infer(bundle, &[0.0; WIDTH])` process at the loader-derived width; bundle bytes and SHA-256 are identical before and after.
- No PR is needed for the 60 s threshold: every packet passes on the current base; closing is your call.
